src/Forms/Shared/Samples/Utility network/DisplayUtilityAssociations/DisplayUtilityAssociations.xaml.cs

```diff
@@ -0,0 +1,155 @@
+
+        // This viewpoint shows several associations clearly in the utility network.
+        private readonly Viewpoint InitialViewpoint = new Viewpoint(new MapPoint(-9812697.41544719, 5131928.28213524, SpatialReferences.WebMercator), 73.4352610787649);
+
+        // Max scale at which to create graphics for the associations.
+        private const double _maxScale = 2000;
+
+        // Symbols for the associations.
+        private Symbol _attachmentSymbol = new SimpleLineSymbol(SimpleLineSymbolStyle.Dot, Color.Green, 5d);
+        private Symbol _connectivitySymbol = new SimpleLineSymbol(SimpleLineSymbolStyle.Dot, Color.Red, 5d);
+
+        // Overlay to hold graphics for all of the associations.
+        private GraphicsOverlay _associationsOverlay;
+
+        private UtilityNetwork _utilityNetwork;
+
+        public DisplayUtilityAssociations()
+        {
+            InitializeComponent();
+            Initialize();
+        }
+
+        private async void Initialize()
+        {
+            try
+            {
+                // Create the utility network.
+                _utilityNetwork = await UtilityNetwork.CreateAsync(new Uri(FeatureServerUrl));
+
+                MyMapView.Map = new Map(Basemap.CreateTopographicVector());
+
+                // Get all of the edges and junctions in the network.
+                IEnumerable<UtilityNetworkSource> edges = _utilityNetwork.Definition.NetworkSources.Where(n => n.SourceType == UtilityNetworkSourceType.Edge);
+                IEnumerable<UtilityNetworkSource> junctions = _utilityNetwork.Definition.NetworkSources.Where(n => n.SourceType == UtilityNetworkSourceType.Junction);
+
+                // Add all edges that are not subnet lines to the map.
+                foreach (UtilityNetworkSource source in edges)
+                {
+                    if (source.SourceUsageType != UtilityNetworkSourceUsageType.SubnetLine && source.FeatureTable != null)
+                    {
+                        MyMapView.Map.OperationalLayers.Add(new FeatureLayer(source.FeatureTable));
+                    }
+                }
+
+                // Add all junctions to the map.
+                foreach (UtilityNetworkSource source in junctions)
+                {
+                    if (source.FeatureTable != null)
+                    {
+                        MyMapView.Map.OperationalLayers.Add(new FeatureLayer(source.FeatureTable));
+                    }
+                }
+
+                // Create a graphics overlay for associations.
+                _associationsOverlay = new GraphicsOverlay();
+                MyMapView.GraphicsOverlays.Add(_associationsOverlay);
+
+                // Populate the legend in the UI.
+                RuntimeImage attachmentSwatch = await _attachmentSymbol.CreateSwatchAsync();
+                AttachmentImage.Source = await Esri.ArcGISRuntime.Xamarin.Forms.RuntimeImageExtensions.ToImageSourceAsync(attachmentSwatch);
```

Comment:
You should be able to, but that returns the platform specific image objects instead of the Xamarin.Forms Image object. Its an API quirk that is related to the import order. I wanted to be explicit with which library I'm using for the `ToImageSourceAsync` call.